webapp/package.json

```diff
@@ -15,60 +16,46 @@
   "author": "",
   "license": "",
   "devDependencies": {
-    "@babel/cli": "^7.8.4",
-    "@babel/core": "7.4.5",
-    "@babel/plugin-proposal-class-properties": "7.4.4",
-    "@babel/plugin-proposal-object-rest-spread": "7.5.5",
-    "@babel/plugin-syntax-dynamic-import": "7.2.0",
-    "@babel/polyfill": "7.4.4",
-    "@babel/preset-env": "7.4.5",
-    "@babel/preset-react": "7.0.0",
-    "@babel/preset-typescript": "7.3.3",
-    "@babel/runtime": "7.4.5",
-    "@types/enzyme": "3.10.4",
-    "@types/jest": "24.0.25",
-    "@types/node": "12.12.24",
-    "@types/react": "16.9.17",
-    "@types/react-dom": "16.9.4",
-    "@types/react-intl": "2.3.18",
-    "@types/react-router-dom": "4.3.4",
-    "@types/react-transition-group": "4.2.2",
-    "@typescript-eslint/eslint-plugin": "1.13.0",
-    "@typescript-eslint/parser": "1.13.0",
-    "babel-eslint": "10.0.2",
-    "babel-jest": "24.9.0",
-    "babel-loader": "8.0.6",
-    "babel-plugin-typescript-to-proptypes": "0.17.1",
-    "compass-mixins": "0.12.10",
-    "css-loader": "3.1.0",
-    "enzyme": "3.11.0",
-    "enzyme-adapter-react-16": "1.15.2",
-    "enzyme-to-json": "3.4.3",
-    "eslint": "6.0.1",
-    "eslint-import-resolver-webpack": "0.11.1",
-    "eslint-plugin-import": "2.18.0",
-    "eslint-plugin-react": "7.14.2",
-    "file-loader": "4.0.0",
-    "identity-obj-proxy": "3.0.0",
-    "jest": "24.9.0",
-    "jest-canvas-mock": "2.2.0",
-    "jest-junit": "10.0.0",
+    "@babel/core": "^7.10.2",
+    "@babel/plugin-proposal-class-properties": "^7.10.1",
+    "@babel/plugin-proposal-object-rest-spread": "^7.10.1",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/preset-env": "^7.10.2",
+    "@babel/preset-react": "^7.10.1",
+    "@babel/preset-typescript": "^7.10.1",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.1.0",
+    "babel-plugin-typescript-to-proptypes": "^1.3.2",
+    "css-loader": "^3.5.3",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "enzyme-to-json": "^3.5.0",
+    "eslint": "^7.1.0",
+    "eslint-import-resolver-webpack": "^0.12.1",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-react": "^7.20.0",
+    "jest": "^26.0.1",
+    "jest-canvas-mock": "^2.2.0",
+    "jest-junit": "^10.0.0",
```

Comment:
Based on the discussion at https://community-daily.mattermost.com/core/pl/hkzq6sunwt8gpr4n14fd6y45bw is seams like exact versions should be used.